### PR TITLE
Update assets, use different sounds, remove minor design flaws

### PR DESCRIPTION
**Pull request by Faffie**
Used inCode to detect design flaws. These changes get rid of the 'Data Class' design flaw in Minigame and the Tradition Breakers in a few (3) classes that inherit from Minigame (overrides with no body).
According to inCode, this (sub)project does not contain any design flaws anymore after this change.

I re-added some music and added a new track to play when the game is lost. Also fixed a few minor style warnings. 
Also updated the splash screen and used a flattened PNG there instead. Added two new events. Restored formatting for a few classes.

### CODE REVIEW

File: EscapeVR/src/main/java/nl/tudelft/kroket/event/events/GameStartEvent.java [hunk 1]
```diff
@@ -4,9 +4,13 @@
 
 public class GameStartEvent extends EventObject {
 
+  /**
+   * 
+   */
+  private static final long serialVersionUID = -3907975018392629402L;
```
Comment (irenevanderblij) [inline EscapeVR/src/main/java/nl/tudelft/kroket/event/events/GameStartEvent.java:10]: Why do all these serialVersions have empty javadoc comments above them?

**Comment by Faffie** [comment]
XD

Because my IDE (Eclipse) adds that when generating an ID.

File: EscapeVR/src/main/java/nl/tudelft/kroket/event/events/GameStartEvent.java (hunk 1 above)
Comment (Faffie) [inline EscapeVR/src/main/java/nl/tudelft/kroket/event/events/GameStartEvent.java:10]: Because my IDE (Eclipse) adds that when generating an ID.

**Comment by Mayke93** [comment]
Looks good according to inCode, I have seen the output.

**Pull request merged**